csharp/Platform.IO/FileHelpers.cs

```diff
@@ -20,6 +48,22 @@ public static T[] ReadAll<T>(string path)
             return reader.ReadAll<T>();
         }
 
+        /// <summary>
+        /// <para>Reads and returns <typeparamref name="T"/> struct from <paramref name="path"/>.</para>
+        /// <para>Считывае, и возвращает структуру типа <typeparamref name="T"/> из <paramref name="path"/>.</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>The structure type.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="path">
+        /// <para>The path to the file, from which to read array of <typeparamref name="T"/> structures.</para>
+        /// <para>Путь к файлу, из которого нужно прочитать массив структур типа <typeparamref name="T"/>.</para>
+        /// </param>
+        /// <returns>
+        /// <para><typeparamref name="T"/> Struct if reading from <paramref name="path"/> was successfull, otherwise default struct.</para>
```

Comment:
`<typeparamref name="T"/> Struct` ⇾ `A <typeparamref name="T"/> struct value`

Reply:
`if reading from <paramref name="path"/> was successfull` ⇾ `if read from <paramref name="path"/> is successful`

Reply:
`otherwise default struct` ⇾ `otherwise the default struct value`